src/layouts/sidebar-sidecar/components/table-of-contents/index.tsx

```diff
@@ -10,12 +10,19 @@ const TABLE_OF_CONTENTS_LABEL_ID = 'table-of-contents-label'
 
 const TableOfContents = ({ headings }: TableOfContentsProps): ReactElement => {
   const { isDesktop } = useDeviceSize()
+  const hasOneHeading = headings.length === 1
+  const enableActiveSection = hasOneHeading ? false : isDesktop
```

Comment:
🎨 Up to you if you want to make this change or not, but I'd personally find the following quite a bit easier to read & understand than a ternary!

```suggestion
  const enableActiveSection = !hasOneHeading && isDesktop
```

Reply:
I'm gonna leave it as is for now 👐